src/scripts/main.js

```diff
@@ -1 +1,106 @@
 'use strict';
+
+const bodyDoc = document.querySelector('body');
```

Comment:
It's a good practice to use `document.body` directly instead of `document.querySelector('body')` since `body` is a standard property on `document`.